nutanix/config.go

```diff
@@ -159,4 +165,5 @@ type Client struct {
 	VmmAPI              *vmm.Client
 	DataPoliciesAPI     *datapolicies.Client
 	LcmAPI              *lcm.Client
+	Calm                *calm.Client
```

Comment:
Can we follow the consistency here? For other modules we have appended a string API to the module name. Do the same for Calm client as well.